### ISSUE TITLE
username/password

### ISSUE THREAD

**Issue by vikrammirla** [daeilkim/refinery#5]
Hello Daeil,
After installing Ubuntu Virtual Box, I tried to access tool at 11:11:11:11:8080. I see a "internal server error".

To debug this I tried to login to the virtual machine, using username/password: doc/refinery. It did not work.
Please advise.
Thanks

**Comment by oolongtea** [comment]
Hi @vikrammirla, I got a similar "Internal Server Error" with the following error:

```
    if g.user is not None and g.user.is_authenticated(): 
TypeError: 'bool' object is not callable
```

This was fixed by editing line 50 of `refinery/refinery/webapp/admin.py` and replacing it with:

```
    if g.user is not None and g.user.is_authenticated:
```

this could have been due to some change in the Flask API (I was using Flask 0.10.1).

**Comment by joaquinvanschoren** [comment]
@oolongtea Thanks this works.

@daeilkim Why are the fixes proposed in #1 and #5 not pushed to the repo?

**Comment by daeilkim** [comment]
Fixes have been made. Thanks for finding those bugs.
